docs/content/upgrade-guides/admin/1-0-0.md

```diff
@@ -0,0 +1,40 @@
+# Medusa Admin: 1.0.0
+
+Medusa Admin has been moved from a standalone repository to our mono-repository to centralise projects and ship it as an NPM package.
```

Comment:
suggestion: is it important to highlight that it was moved to our mono-repository? If you think it is, then I'm fine with keeping it. If not, I suggest just focusing that it's now an NPM package that can be installed on the backend. Something like this:

```suggestion
Medusa Admin has been moved from being hosted on a GitHub repository to being published as an NPM package that can be installed in the same project as the Medusa backend. This upgrade guide will help you find the right resources to make the upgrade.
```